nginx/assets/docs/spec.yaml

```diff
@@ -0,0 +1,259 @@
+
+                To get more detailed metrics with NGINX Plus (such as 2xx / 3xx / 4xx / 5xx response counts per second), set a `status_zone` on the servers you want to monitor. For example:
+
+                ```conf
+                server {{
+                  listen 80;
+                  status_zone <ZONE_NAME>;
+                  ...
+                }}
+                ```
+
+                Reload NGINX to enable the status or API endpoint. There's no need for a full restart.
+
+                ```shell
+                sudo nginx -t && sudo nginx -s reload
+                ```
+
+
+                [5]: https://nginx.org/en/docs/http/ngx_http_api_module.html
+      - template: setup/configuration
+        overrides:
+          description: |
+
+          sections:
+            - template: setup/configuration/host
+              overrides:
+                description: |
+                  To configure this check for an Agent running on a host:
+
+                  Follow the instructions below to configure this check for an Agent running on a host. For containerized environments, see the [Containerized section](#containerized).
+              sections:
+              - name: Metric collection
+                header_level: 4
+                description: |
+                  1. Set the `nginx_status_url` parameter to `http://localhost:81/nginx_status/` in your `nginx.d/conf.yaml` file to start gathering your [NGINX metrics](#metrics). See the [sample nginx.d/conf.yaml][6] for all available configuration options.
+
+                      **NGINX Plus**:
+
+                        - For NGINX Plus releases 13+, set the parameter `use_plus_api` to `true` in your `nginx.d/conf.yaml` configuration file.
+                        - Stream stats API calls are included by default for NGINX Plus. If you want to disable them, set the parameter `use_plus_api_stream` to `false` in your `nginx.d/conf.yaml` configuration file.
+                        - If you are using `http_api_module`, set the parameter `nginx_status_url` to the server's `/api` location in your `nginx.d/conf.yaml` configuration file, for example:
+
+                            ```yaml
+                            nginx_status_url: http://localhost:8080/api
+                            ```
+
+                  2. Optional - If you are using the NGINX `vhost_traffic_status module`, set the parameter `use_vts` to `true` in your `nginx.d/conf.yaml` configuration file.
+
+                  3. [Restart the Agent][7] to start sending NGINX metrics to Datadog.
+
+                  [6]: https://github.com/DataDog/integrations-core/blob/master/nginx/datadog_checks/nginx/data/conf.yaml.example
+                  [7]: https://docs.datadoghq.com/agent/guide/agent-commands/#start-stop-and-restart-the-agent
+              - template: setup/configuration/log_collection
+                overrides:
+                  prepend_text: |
+                    _Available for Agent versions >6.0_
```

Comment:
Good use of `prepend_text` 😄  - but I think we are starting to drop this clause now.